### PR TITLE
NGINX Config - Add info about remote storages (S3, GCS)

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
Add info about remote storages like S3 or GCS, the nginx location cannot handle that.

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 1]
```diff
@@ -407,6 +407,9 @@ server {
 
     # Assets
     # Still use a whitelist approach to prevent each and every missing asset to go through the PHP Engine.
+    # If you are using remote storages like S3 or Google Cloud Storage, this doesn't work. You either deactivate it and handle it in PHP
+    # or redirect these suffixes directly to your CDN URL.
+    location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif
     location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif|png|svgz?|eps|exe|gz|zip|mp\d|m4a|ogg|ogv|webm|pdf|docx?|xlsx?|pptx?)$ {
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]: This doesn't look right.

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 2]
```diff
@@ -407,6 +407,9 @@ server {
 
     # Assets
     # Still use a whitelist approach to prevent each and every missing asset to go through the PHP Engine.
+    # If you are using remote storages like S3 or Google Cloud Storage, this doesn't work. You either deactivate it and handle it in PHP
+    # or redirect these suffixes directly to your CDN URL.
```
Comment (brusch) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]:
```suggestion
    # or redirect these suffixes directly to your CDN URL. Additionally you should configure the frontend url prefixes accordingly, see: https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html
```

**Review by brusch — CHANGES_REQUESTED** [review]

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md (hunk 1 above)
Comment (dpfaffenbauer) [reply doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]: fixed

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md [hunk 3]
```diff
@@ -407,6 +407,9 @@ server {
 
     # Assets
     # Still use a whitelist approach to prevent each and every missing asset to go through the PHP Engine.
+    # If you are using remote storages like S3 or Google Cloud Storage, this doesn't work. You either deactivate it and handle it in PHP
+    # or redirect these suffixes directly to your CDN URL. Additionally you should configure the frontend url prefixes accordingly, see: https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html
+    # location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif?)$ {
     location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif|png|svgz?|eps|exe|gz|zip|mp\d|m4a|ogg|ogv|webm|pdf|docx?|xlsx?|pptx?)$ {
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]: I don't get it. What is this commented `location` line for? Why does it only contain some of the types below?

Comment (dpfaffenbauer) [reply doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md]: true, something went wrong there. I removed the line. Idea is, that, if you use remote storages, nginx can never resolve these files. In our Cloud Kubernetes Setup, we completely commented the location handle out.

**Review by kingjia90 — COMMENTED** [review]
Might need to be added to the Development Enviroment sample nginx as well https://github.com/dpfaffenbauer/pimcore/blob/34f2f854868dde22e0f5155b6ae4fbfb3b4ec0d1/doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md?plain=1#L125

and preferably add a new line in this list of "assumptions", something like "Assets are not stored in remote storages; If so, please see the comments under the block `# Assets`"

https://github.com/dpfaffenbauer/pimcore/blob/34f2f854868dde22e0f5155b6ae4fbfb3b4ec0d1/doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md?plain=1#L11-L17

**Comment by dpfaffenbauer** [comment]
@kingjia90 done

**Pull request merged**